spec/pubid_iso/identifier/amendment_spec.rb

```diff
@@ -92,6 +92,24 @@ module Identifier
               expect(subject.to_s).to eq("ISO #{number}:1999/DAM 1")
             end
           end
+
+          context "when stage is a harmonized code" do
+            context "when stage is 50.00" do
+              let(:stage) { "50.00" }
+
+              it "renders long stage and amendment" do
+                expect(subject.to_s).to eq("ISO #{number}:1999/FDAM 1")
+              end
+            end
+
+            context "when stage is 60.60" do
```

Comment:
@mico can you add a test for "60.00"? The result should also be `ISO #{number{:1999}/Amd 1`.